src/main/java/address/updater/DependencyTracker.java

```diff
@@ -130,6 +129,7 @@ private void excludePlatformSpecificDependencies(List<String> dependencies) {
 
         try {
             updateData = JsonUtil.fromJsonString(json, UpdateData.class);
+            //TODO: is it possible to have a generic static method in StorageManager to read/write various json files?
```

Comment:
Should I replace all Json file handling in this PR or a separate PR? If separate PR, I will do replace all Json file handling in updater first.